modules/ROOT/pages/afm-create-monitor.adoc

```diff
@@ -49,42 +49,49 @@ For details, see xref:afm-monitoring-public-apis.adoc[Monitoring the Endpoints o
 == Procedure
 To create a monitor:
 
-. In the *Functional Monitoring* section of Anypoint Monitoring, click *Create*.
-. Specify a name and a short description for the monitor.
-. Fill in the fields with the information that you have gathered.
-. Click *Create*.
+. In the *Functional Monitoring* section of Anypoint Monitoring, click *Create Monitor* in the top-right corner of your browser window.
+. In the *Monitor Setup* section, specify the name of the monitor, the location from which to run the test, how often to run the test, and whether to validate SSL certificates.
+.. Specify a name for the monitor.
+.. Select the location from which to run the test.
++
+Public locations are grouped separately from private locations in the list. You can create a private location by clicking an option in the list.
+.. Specify the intervals at which you want the monitor to run the test.
++
+The interval that you select begins when you click *Create Monitor* at the end of this procedure.
+.. Toggle the *SSL Certificate Validation* switch off if the endpoints that your monitor will test use self-signed certificates.
++
+If they do use self-signed certificates and you do not toggle this switch off, tests against those endpoints fail.
++
+** If the toggle is on, all endpoints that use SSL certificates must have certificates that are signed by a trusted authority.
+** If the toggle is off, endpoints that use SSL certificates may be self-signed, but do not have to be.
```

Comment:
See above. Trying to simplify this, but now that I see the following paragraph, I don't know if that is possible.
It seems to me that this is too much conceptual information that is not part of the focus of this task, and perhaps should be covered elsewhere (about endpoint types and settings).
Or maybe we could just omit the following info about testing endpoints, as that is somewhat outside of the obligation, for THIS task, to select a switch setting.